modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate the connector, the connector is upgraded to run on Mule runtime 4.6. and TLS is enabled so that Anypoint Studio displays the appropriate form to collect required TLS connection configuration information for the connector. 
```

Comment:
I suggest using bullets and tweaking the wording to break up the long text.

When you enable TLS support and generate a connector:

*  The connector is upgraded to run on Mule runtime 4.6. 
*  TLS is enabled, so that Anypoint Studio displays the appropriate form for collecting the connector's required TLS connection configuration information.